### PR TITLE
Release 0.11.1

### LINKED ISSUE #145: The number of samples after update is incorrect
image

### PR DESCRIPTION
**Pull request by khoroshevskyi**
## Changes:
- Added namespace tar metadata table and methods
- Fixed #145

### CODE REVIEW

File: pepdbagent/models.py [hunk 1]
```diff
@@ -293,3 +293,25 @@ class SchemaGroupSearchResult(BaseModel):
     limit: int
     offset: int
     results: List[SchemaGroupAnnotation]
+
+
+class TarNamespaceModel(BaseModel):
+    """
+    Geo project model
```
Comment (donaldcampbelljr) [inline pepdbagent/models.py:300]: is this description and the one below it accurate?

File: pepdbagent/modules/namespace.py [hunk 2]
```diff
@@ -300,3 +302,73 @@ def stats(self, namespace: str = None, monthly: bool = False) -> NamespaceStats:
...
+
+        :return: None
+        """
+
+        with Session(self._sa_engine) as session:
+
+            delete_statement = delete(TarNamespace)
+            if namespace:
+                delete_statement = delete_statement.where(TarNamespace.namespace == namespace)
+            session.execute(delete_statement)
```
Comment (donaldcampbelljr) [inline pepdbagent/modules/namespace.py:372]: How does deletion work if there is no namespace given?

**Review by donaldcampbelljr — DISMISSED** [review]
minor questions but it looks good overall

File: pepdbagent/modules/namespace.py (hunk 2 above)
Comment (khoroshevskyi) [reply pepdbagent/modules/namespace.py:372]: It will just delete all tars from all namespaces

File: pepdbagent/models.py (hunk 1 above)
Comment (khoroshevskyi) [reply pepdbagent/models.py:300]: no, Good catch!

**Review by donaldcampbelljr — APPROVED** [review]

**Pull request merged**